uploader.html

```diff
@@ -110,183 +89,76 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
           </div>
 
           <div class="modal-body">
-            <div id="select-file-section" class="row"></div>
-            <div id="media-data-section" class="row col-sm-8 pull-right"></div>
-            <div id="selected-media-section" class="row col-sm-12"></div>
-
-            <!-- select new file -->
-            <div id="js-select-file-section">
-              <script type="text/html">
-                <div class="select-label col-sm-4">
-                  Select new <%= filetype %>
-                </div>
-                <div class="select-btn col-sm-4">
-                  <label for="js-file-upload" id="js-file-upload-label"class="select-file button btn-sm">
-                    Select File
-                  </label>
-                  <input id="js-file-upload" type="file" accept="image/*" capture="camera" style="visibility:hidden;">
-                </div>
-              </script>
+            <div class="row">
+              <div id="select-file-section"></div>
             </div>
-              
-              <!-- create data, progress, ready text container -->
-            <div id="js-media-data-section" class="container">
-              <script type="text/html">
-                <div id="js-file-data" class="file-data row col-sm-12">
-                  <span class="glyphicon glyphicon-ok"></span> <%= fileName %> [<%= fileSize %>] MB
-                </div>
-                <div id="js-file-progress" class="row">
-                  <progress class="col-sm-11" value="10000" max="<%= fileSize %>"></progress>
-                </div>
-                <div id="js-file-ready-text" class="ready-text row">
-                  Ready to upload. Click <strong>Begin Upload</strong> to start.
-                </div>
-                </script>
+            <div class="row">
+              <div id="selected-media-section"></div>  
             </div>
-
-            <!-- selection section container -->
-            <div id="js-selected-media-section">
-              <script type="text/html">
-                <div id="js-current-label" class="js-current-label col-sm-4">
-                  Current <%= filetype %>
-                </div>
-                <div id="js-current-preview" class="js-current-preview col-sm-4">
-                  <img id="js-output-file"/>
-                </div>
-                <div id="js-current-btn" class="col-sm-4">
-                  <button id="js-full-image" class="button btn-sm">View Full Size</button>
-                </div>
-              </script>
+            <div class="row">
+              <div id="media-data-section"></div>
             </div>
           </div>
 
           <div class="modal-footer">
-            <!-- <span class="glyphicon glyphicon-cloud-upload"></span> -->
+            <button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
+            <!-- <i class="fa fa-cloud-upload"></i> -->
```

Comment:
dead code, should either uncomment or remove

Reply:
Actually, I meant to move this into the `button` tag so that the icon shows on the button.  Have done that now.